legendary/downloader/mp/manager.py

```diff
@@ -166,6 +167,16 @@ def run_analysis(self, manifest: Manifest, old_manifest: Manifest = None,
             mc.changed -= files_to_skip
             mc.unchanged |= files_to_skip
 
+        if file_exclude_filelist:
+            file = open(file_exclude_filelist, 'r')
+            file_exclusions = set(file.read().splitlines())
+            file.close()
```

Comment:
I realize that reading the file directly from the downloader might be a bad idea, would it suffice to move them in Core or CLI perhaps?